src/components/github-contributors-list/github-contributors-list.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, useEffect, useMemo, useState } from 'react';
+import ContributorsList from './contributors-list';
+import Paragraph from '../../../src/components/paragraph/paragraph';
+import { getAllContributors, GithubContributor, GithubContributorResponse } from './github-contributors-utils';
+
+interface GithubContributorsListProps {
+  organizationName: string;
+  packageName: string;
+  text?: string;
+  showContributionAmount?: boolean;
+  excludedContributorsIds?: Set<number>;
+  staticContributors?: GithubContributor[];
+}
+
+const GithubContributorsList: FC<GithubContributorsListProps> = ({
+  organizationName,
+  packageName,
+  excludedContributorsIds = new Set(),
```

Comment:
I think this can be externally and removed from this component, WDYT?

Reply:
You mean `excludedContributorsIds` prop in general? Or it's default value? Why?

Reply:
Oops sorry commented on the wrong line, I meant for the text prop